pom.xml

```diff
@@ -35,40 +35,38 @@
-        <azure-bom-version>1.2.18</azure-bom-version>
+        <azure-bom-version>1.2.22</azure-bom-version>
 
         <!-- Versions below are for several dependencies we're using in the data & ingest modules -->
         <!-- Ideally, versions below should align with latest databricks runtime dependency versions -->
         <!-- Compile dependencies -->
         <slf4j.version>1.7.36</slf4j.version>
         <commons-lang3.version>3.14.0</commons-lang3.version>
         <commons-text.version>1.11.0</commons-text.version>
-        <commons-codec.version>1.16.0</commons-codec.version>
-        <msal4j.version>1.13.10</msal4j.version>
+        <commons-codec.version>1.16.1</commons-codec.version>
+        <msal4j.version>1.14.3</msal4j.version>
         <apache.httpclient.version>4.5.14</apache.httpclient.version>
         <httpcore.version>4.4.16</httpcore.version>
-        <fasterxml.jackson.core.version>2.15.2</fasterxml.jackson.core.version>
-        <reactor-core.version>3.4.34</reactor-core.version>
+        <fasterxml.jackson.core.version>2.16.1</fasterxml.jackson.core.version>
         <univocity-parsers.version>2.9.1</univocity-parsers.version>
         <resilience4j.version>1.7.1</resilience4j.version>
-        <io.vavr.version>0.10.2</io.vavr.version>
+        <io.vavr.version>0.10.4</io.vavr.version>
         <!-- Test dependencies -->
         <bouncycastle.version>1.77</bouncycastle.version>
         <jsonassert.version>1.5.0</jsonassert.version>
-        <sqlite-jdbc.version>3.44.1.0</sqlite-jdbc.version>
-        <annotations.version>22.0.0</annotations.version>
+        <sqlite-jdbc.version>3.45.2.0</sqlite-jdbc.version>
+        <annotations.version>24.1.0</annotations.version>
 
         <!-- Other dependencies -->
         <maven-compiler-plugin.version>3.8.1</maven-compiler-plugin.version>
-        <maven-assembly-plugin.version>3.3.0</maven-assembly-plugin.version>
-        <maven-surefire-plugin.version>3.2.1</maven-surefire-plugin.version>
-        <maven-javadoc-plugin.version>3.2.0</maven-javadoc-plugin.version>
-        <maven-source-plugin.version>3.2.1</maven-source-plugin.version>
-        <maven-dependency-plugin.version>3.1.2</maven-dependency-plugin.version>
-        <flatten-maven-plugin.version>1.2.5</flatten-maven-plugin.version>
-        <junit.version>5.8.2</junit.version>
-        <mockito.version>5.7.0</mockito.version>
+        <maven-assembly-plugin.version>3.7.1</maven-assembly-plugin.version>
+        <maven-surefire-plugin.version>3.2.5</maven-surefire-plugin.version>
+        <maven-javadoc-plugin.version>3.6.3</maven-javadoc-plugin.version>
+        <maven-source-plugin.version>3.3.0</maven-source-plugin.version>
+        <maven-dependency-plugin.version>3.6.1</maven-dependency-plugin.version>
+        <flatten-maven-plugin.version>1.2.7</flatten-maven-plugin.version>
+        <junit.version>5.10.2</junit.version>
+        <mockito.version>5.11.0</mockito.version>
```

Comment:
Please correct me here :  did mockito not move out of JDK 8 to JDK 11. There is some diff in inline and normal mocks is it perhaps that ? (or) do we even use Mockito if it does not give us that error of 
"This library is compiled with Version 55 and will not work with 52"

Ref : https://github.com/mockito/mockito/releases/tag/v5.0.0 

_What should I do if I still run JDK 8?
For JDK 8 and below, you can keep on using Mockito 4._

Reply:
@ag-ramachandran I believe you are correct. Directly beneath this line in the POM file (lines 72 to 81) it looks like someone set JDK 8 and 11 builds to utilize Mockito 4.5.1 whereas I suppose the other builds (17 and 21) use this dependency.